src/server/gc_worker/gc_worker.rs

```diff
@@ -500,6 +509,14 @@ where
                     limit,
                 );
             }
+            GcTask::OrphanVersions(wb) => {
+                info!("handling GcTask::OrphanVersions"; "size" => wb.as_inner().count());
```

Comment:
can we give the message an ID and print both in compaction filter and here? so in the log we can match them.